phys/module_sf_clm.F

```diff
@@ -60994,7 +60994,7 @@ subroutine clminit(VEGFRA,SNOW,SNOWC,SNOWH,CANWAT,SMSTAV,        &
      ENDDO
    ENDDO
    IF ( errflag .EQ. 1 ) THEN
-      CALL wrf_error_fatal( "CLM: clminit: out of range value "// &
+      CALL wrf_error_fatal( "CLM: clminit: out of range value ", &
```

Comment:
What is the reason for this change?

Reply:
I think we've typically used flags like `-P -traditional` with `cpp` to avoid various issues when processing Fortran. Given that `//` is a standard concatenation operator in Fortran, rather than forevermore having to ensure that new code doesn't make use of this, it seems better to adjust our preprocessing if at all possible.

Reply:
I see, I think that flag slipped past me, as I had `-P -nostdinc` but not `-traditional` added to my `CMAKE_C_PREPROCESSOR_FLAGS` @ line 25 at the top-level CMakeLists.txt